Atomic/containers.py

```diff
@@ -134,13 +134,23 @@ def ps_tty(self):
 
 
         container_objects = self._ps()
-        if len(container_objects) == 0:
-            return 0
+        # If we were not asked for json output, return out with no output
+        # when there are no applicable container objects
+        if not self.args.json:
+            if len(container_objects) == 0:
+                return 0
 
-        if not any([x.running for x in container_objects]) and not self.args.all:
-            return 0
+            if not any([x.running for x in container_objects]) and not self.args.all:
+                return 0
+
+        # Set to 12 when truncate
+        if self.args.truncate:
```

Comment:
Whats the purpose of expanding the short form of the expression previously used? Did it trigger a failure or doing for readability?